services/api/src/utils/emails.js

```diff
@@ -25,23 +52,38 @@ async function initialize() {
   await Promise.all(
     files.map((file) => {
       return readFile(path.join(templatesDist, file)).then((str) => {
-        templates[file] = str.toString();
+        if (file.includes('.md')) {
+          templates[file] = marked(str.toString());
+        } else {
+          templates[file] = str.toString();
+        }
       });
     })
   );
   isReady = true;
```

Comment:
I know you didn't change this part and just an idea but for this kind of initialization pattern I like to use `once` from lodash:

```
const { once } = require('lodash');
const initialize = once(async () => {
   // do some async stuff
});
```

could get rid of all the`isReady` stuff that way

Reply:
actually in this case however you could actually just initialize the module inline when it's required... ie:

```
initialize();
module.exports = { ... }
```

Reply:
actually re-re thinking it, it might not be impossible for an instance to fire up and immediately try to send out an email which could fail if not initialized yet so maybe the `once` pattern is simpler after all... either that or make `initialize` synchronous instead

Reply:
Yeah let me look at that. 
I think we should also consider that the markdown might be stored in a mongo instead on disk (slightly in the future), so i am thinking i should keep it async.

Reply:
ah true so it might need an async startup anyway... but we could also come back to that when it happens so feel free to just ignore me here too